src/resources/swagger.yaml

```diff
@@ -467,15 +467,6 @@ paths:
       responses:
         '200':
           description: OK
-          content:
-            application/json:
```

Comment:
Technically, this returns `text/plain` (I think?) since you have a string response. This is me being pedantic though, swagger won't care.